mlflow/genai/judges/utils.py

```diff
@@ -404,6 +437,9 @@ def invoke_judge_model(
         assessment_name: The name of the assessment.
         trace: Optional trace object for context.
         num_retries: Number of retries on transient failures when using litellm.
+        output_format: Optional Pydantic model class for structured output format.
+                       If not provided, defaults to the standard Judge output format
+                       with 'result' and 'rationale' fields.
 
```

Comment:
@BenWilson2 Can we move this to `_invoke_litellm` instead? Judges should always return `value` / `rationale`. It's also a bit strange if the request / response field extractor returns a `Feedback` instance.